app/src/views/PerProcessForm/index.tsx

```diff
@@ -61,35 +61,34 @@ export function Component() {
         ?? getCurrentPerProcessStep(statusResponse)
         ?? PER_PHASE_OVERVIEW;
 
+    const contentOriginalLanguage = statusResponse
+        ?.translation_module_original_language;
+    const nonEnglishCreate = isNotDefined(perId) && currentLanguage !== 'en';
```

Comment:
@frozenhelium Do we need this now? Shouldn't we allow the users to create PER in all the languages?

Reply:
@samshara That's correct! I've removed it.